pkg/cri/config/config.go

```diff
@@ -77,11 +77,11 @@ type Runtime struct {
 	// while using default snapshotters for operational simplicity.
 	// See https://github.com/containerd/containerd/issues/6657 for details.
 	Snapshotter string `toml:"snapshotter" json:"snapshotter"`
-	// SandboxMode defines which sandbox runtime to use when scheduling pods
+	// Sandboxer defines which sandbox runtime to use when scheduling pods
 	// This features requires experimental CRI server to be enabled (use ENABLE_CRI_SANDBOXES=1)
 	// shim - means use whatever Controller implementation provided by shim (e.g. use RemoteController).
 	// podsandbox - means use Controller implementation from sbserver podsandbox package.
-	SandboxMode string `toml:"sandbox_mode" json:"sandboxMode"`
+	Sandboxer string `toml:"sandboxer" json:"sandboxer"`
```

Comment:
Maintain the previous key as well, but set it as deprecated. We will deal with the removal of deprecated keys in a consistent way.

Reply:
@dmcgowan How do we deal with this config key? is it option 2 above?